packages/containerd-2.0/containerd-2.0.spec

```diff
@@ -0,0 +1,176 @@
+%global goproject github.com/containerd
+%global gorepo containerd
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 2.0.5
+%global rpmver %{gover}
+%global gitrev fb4c30d4ede3531652d86197bf3fc9515e5276d9
+
+%global _dwz_low_mem_die_limit 0
+
+Name: %{_cross_os}%{gorepo}-2.0
+Provides: %{_cross_os}%{gorepo} = 0
+Conflicts: %{_cross_os}%{gorepo}-1.7
```

Comment:
I've seen that we default to `Provides` and `Conflicts` closer to the `Requires`  in the specs. Personally I prefer that because it is simpler to read the bulk of dependencies.